src/doc/trpl/concurrency.md

```diff
@@ -215,29 +220,15 @@ fn main() {
 }
 ```
 
-
-If we'd tried to use `Mutex<T>` without wrapping it in an `Arc<T>` we would have
```

Comment:
The above code snippet without `Arc` will look like [this](http://is.gd/COPLRq), which gives a "capture of moved value" error...